tests/unit/test_network_connections.py

```diff
@@ -437,6 +438,8 @@ def test_validate_dict(self):
             {"i": 5, "s": "s_default", "l": "6"}, v.validate({"i": "5", "l": "6"})
         )
         self.assertValidationError(v, {"k": 1})
+        # Assert that `None` is normalized into `{}`` in ArgvalidatorDict()
+        self.assertEqual({}, v.validate(None))
```

Comment:
Please switch the arguments, the expected value should be on the right-hand side.

Reply:
ok, the reason that I written this way is because I saw some other testcases in `test_network_conenctions.py` put the expected value on the left handside to `self.assertEqual()`. Nevertherless, I will update.

Reply:
Oh, I did not notice that it is not uniform. If you like, please create a different PR/commit once this is done to make this more uniform.

Reply:
Updated. Thanks.